packages/react-dev-utils/launchEditor.js

```diff
@@ -256,6 +264,10 @@ function launchEditor(fileName, lineNumber) {
     return;
   }
 
+  // colNumber is optional, but should be a number
+  // default is 1
+  colNumber = parseInt(colNumber, 10) || 1;
```

Comment:
Is it not a number by this point? Can we assume it's always either missing or a number?

I would feel better if we forced line and column to be valid numbers in this function and threw if they're not. And validate/parse arguments in the middleware instead.

Reply:
In the middleware, we directly pass line and column from the query string. So they are strings. But I agree with you. Let's validate/parse in the middleware. We can expect positive integers(number might be a decimal or negative integer) for this function and throw an error otherwise. Or do we need that? since this function is only used internally. 🤔

Reply:
Fixed with https://github.com/facebookincubator/create-react-app/pull/3465/commits/2e346d4d763d1bbbf9122398a2e1827a194f3a2e